pkg/process/config/config.go

```diff
@@ -151,6 +151,12 @@ func NewDefaultAgentConfig() *AgentConfig {
 	//       need a few minutes to be ready.
 	_, err = util.GetContainers()
 	canAccessContainers := err == nil
+	canAccessContainers = false
+
+	enabledChecks := make([]string, 0)
+	if canAccessContainers {
```

Comment:
`canAccessContainers` is always false per the above line.  Is that right?

Reply:
Nice catch! I had set this up for testing and accidentally commited it.  I'll remove now.